static/docs/commands-reference/run.md

```diff
@@ -195,3 +206,29 @@ specified, so a `model.p.dvc` stage file is created:
     │   └── test.dvc
     └── test.txt
 ```
+
+* Difference between `run --outs` and `run --outs-persist`
+
+```dvc
```

Comment:
let's break it into two parts and give at least some brief explanation of what's happening.